pkg/module_manager/module_manager_hooks.go

```diff
@@ -15,15 +15,22 @@ import (
 	sh_op_types "github.com/flant/shell-operator/pkg/hook/types"
 )
 
-func (mm *ModuleManager) loadGlobalValues() ( /* global values */ utils.Values /* enabled modules */, map[string]struct{}, error) {
+type globalValues struct {
+	globalValues   utils.Values
+	enabledModules map[string]struct{}
+	configBytes    []byte // config-values.yaml
```

Comment:
NIT: I guess we can rename it to `configSchema` or something like that. Because name `...Bytes` is not informative

Reply:
Fixed